zmon_worker_monitor/zmon_worker/tasks/main.py

```diff
@@ -1090,7 +1090,12 @@ def _store_check_result(self, req, result):
         self.con.sadd('zmon:checks', req['check_id'])
         self.con.sadd('zmon:checks:{}'.format(req['check_id']), req['entity']['id'])
         key = 'zmon:checks:{}:{}'.format(req['check_id'], req['entity']['id'])
-        value = json.dumps(result, cls=JsonDataEncoder)
+        value = "NONE"
+        try:
+            value = json.dumps(result, cls=JsonDataEncoder)
+        except Exception, e:
+            self.logger.exception("failed to serialize check result for check %s", req['check_id'])
+            value = "Serialization error: {}".format(e)
```

Comment:
General nitpicking comment 😄 :
We need to stick to single quotes for strings unless we need it inside the string, so we switch to double quotes. This is not covered in any automated way but it is part of [pep8](https://www.python.org/dev/peps/pep-0008/#string-quotes) nevertheless and it is just muscle memory over time 😉